grpc/src/main/java/com/avast/metrics/grpc/MetricsCache.java

```diff
@@ -28,6 +29,11 @@ public <ReqT, RespT> Meter getMeter(MethodDescriptor<ReqT, RespT> methodDescript
         return meters.computeIfAbsent(methodMonitorName + name, (s) -> monitor.named(methodMonitorName).newMeter(name));
     }
 
+    public <ReqT, RespT> Meter getMeter(MethodDescriptor<ReqT, RespT> methodDescriptor, String name1, String name2) {
+        String methodMonitorName = MetricNaming.getMethodMonitorName(methodDescriptor);
+        return meters.computeIfAbsent(methodMonitorName + name1 + name2, (s) -> monitor.named(methodMonitorName, name1).newMeter(name2));
+    }
```

Comment:
Just did it this way - the `restOfNames` handling would be really awkward otherwise due to this being Java and the variable arguments handling and nulls. The variant with `String... restOfNames` can still be added when needed.

Also, sorry for the push spam, I apparently can't test this locally, looks like even java 11 is "too new".